### DISCUSSION TITLE
G7 won't connect on Pixel 7

### DISCUSSION THREAD

**Discussion by cascer1** [NightscoutFoundation/xDrip#3670] [Bug reports] [answered]
Hi,

I switched to the G7 a few weeks ago. My first two sensors worked fine without any issues, but now I'm trying all I can to connect to my next sensor and it's just not working.

**App version:** dfcbe80-2024.09.17  
**Android version:** 14

Here's what I did:

- About 45 minutes before the current sensor would end, I inserted the new sensor. 
- I waited until the current sensor expired and then tried connecting to the new sensor by changing the Transmitter ID settings to the new value.
- For a while, I got the mismatch error (I forget the exact wording), so placed the old sensor in the microwave. 
- I kept getting the mismatch error, so I walked to a public trash bin on another street to dispose of the old sensor there.
- The mismatch error went away, now it just says `Hunting Transmitter: Stay on this page`. And has been for at least an hour.
- I re-installed xDrip, including older versions and the most recent nightly I could find
- Still in all versions, it just stays stuck on hunting transmitter

**to recap, everything I've tried:**
- restart my phone
- double-check all the settings from the guides online
- re-instlal xDrip
- move far away from the old sensor
- ensure I'm on the latest nightly

What am I doing wrong? I followed all the steps on [the G7 guide](https://navid200.github.io/xDrip/docs/Dexcom/G7.html) and [the connectivity troubleshooting guide](https://navid200.github.io/xDrip/docs/Connectivity-troubleshoot.html)

**Comment by Navid200** [comment, marked as answer]
You don't also have Dexcom app installed collecting from the same device, do you?

Are you sure the transmitter ID has no typo?

Can you use a scanner to confirm that the transmitter is alive and transmitting?
https://navid200.github.io/xDrip/docs/Bluetooth-Scanner.html  

If you see no signal and conclude that something is wrong with G7, you will need to install Dexcom app for 100% confirmation.  You should never contact Dexcom just because xDrip does not work.

**Reply by cascer1** [reply]
Hi, sorry for my untimely response. I followed your instructions and concluded that there must've been something wrong with the sensors. They didn't show up in the ble scanner nor the dexcom app (which I installed later, after removing xDrip). I contacted Dexcom and they replaced both. A third sensor worked immediately with xDrip.
